lib/history.spec.js

```diff
@@ -0,0 +1,9 @@
+
+describe('History', () => {
```

Comment:
Can it be moved to a specific folder please ? 
I'd prefer all the tests to be in a folder `test` and have the same name as the file they're testing.